### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pnijhara**
Project proposal for #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # webSCAS
 Web application for sepsis clinical assessment scores using batch ICU patient data
+Backend of the application will be in django and the project will be completely python based
```
Comment (duddu-coder) [inline README.md]:
```suggestion
Backend of the application will be in Flask and the project will be completely python based
```

**Review by duddu-coder — CHANGES_REQUESTED** [review]
@pnijhara Well, django is a bit heavy for already memory intensive applications. We should consider using Flask for it. 

Have Milk,
Thanks :)

**Comment by duddu-coder** [comment]
> Project proposal

So this is your project proposal. 
Seems nice to me. I would approve that ;)

**Comment by pnijhara** [comment]
@akram-mohammed please review it.

**Pull request merged**